### DISCUSSION TITLE
Do CPM-live support lr change during training

### DISCUSSION THREAD

**Discussion by Kunlun-Zhu** [OpenBMB/CPM-Live#10] [Q&A 问答] [answered]
We could try restore the model at certain checkpoint during training and change its lr or other hyper-parameters manually to prevent overfitting or unconvergence.

**Comment by jayzzhou-thu** [comment, marked as answer]
Thanks for your suggestion! Now the model is stored every 500 steps and the hyper-parameters can be tuned if the training is unstable.
